src/setup-tests.ts

```diff
@@ -2,5 +2,4 @@
 import * as matchers from 'jest-extended'
 expect.extend(matchers)
 
-jest.setTimeout(10000)
```

Comment:
removing the timeout can be dangerous; unless the CI already have a timeout.

Reply:
I moved it to the separate config files so we can have different values depending on how you're executing it (just unit, just integration, or combined)